documentation/metrics.md

```diff
@@ -0,0 +1,78 @@
+## Metrics
+
+It is common for services registering callbacks with the status service to include some basic metrics about the health
+of the service.  These metrics are typically only available at the `:debug` status level, and should be limited to a
+fairly small set of data that can be useful for debugging the service's performance or behavior.
```

Comment:
Would it be good to mention that these have typically been labeled as "experimental"/how we want to version them?